javaparser-core/src/main/java/com/github/javaparser/javadoc/description/JavadocInlineTag.java

```diff
@@ -87,7 +87,7 @@ public JavadocInlineTag(Type type, String content) {
 
     @Override
     public String toText() {
-        throw new UnsupportedOperationException();
+        return "{@" + type.keyword + this.content;
```

Comment:
No, sorry, I meant that I am afraid I could not do the parsing correctly and maybe the brace end up inside the content, while it should not.